core/chains/evm/forwarders/orm.go

```diff
@@ -66,3 +67,17 @@ func (o *orm) FindForwarders(offset, limit int) (fwds []Forwarder, count int, er
 	}
 	return
 }
+
+// FindForwardersByChain returns all forwarder addresses for a chain.
+func (o *orm) FindForwardersByChain(evmChainId utils.Big) (fwds []Forwarder, count int, err error) {
+	sql := `SELECT count(*) FROM evm_forwarders where evm_chain_id = $1`
```

Comment:
Maybe someone else from Core team can comment on this.
Do we want to create a new table here, or insert forwarders into existing evm_chains or similar table.

Reply:
A chain can have many forwarders, so I don't think we want it on `evm_chains`. I'd vote for its own table because it is a new resource and it can have relationships with other objects, like allowed senders and feeds that expect it.